packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -47,6 +47,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
+BuildRequires: %{?scl_prefix}npm(@patternfly/react-catalog-view-extension) >= 4.8.126
+BuildRequires: %{?scl_prefix}npm(@patternfly/react-catalog-view-extension) < 5.0.0
```

Comment:
```
DEBUG util.py:600:  No matching package to install: 'npm(@patternfly/react-catalog-view-extension) >= 4.8.126'
DEBUG util.py:600:  No matching package to install: 'npm(@patternfly/react-catalog-view-extension) < 5.0.0'
```

needs packaging

Reply:
https://github.com/theforeman/foreman-packaging/pull/6671

I wonder tho, if this shouldn't be exposed via `@theforeman/vendor` cc @theforeman/ui-ux

Reply:
I think we went for a model where plugins could still add their own dependencies.

Reply:
Yeah, not saying it *has* to, just wondering if having it there and consumable by everyone wouldn't be easier.

Reply:
+1 for adding it to foreman vendor

Reply:
+1 to adding it as well.  However, I'm fine keeping it here for the time being and then removing it once it's in @theforeman/vendor, since there's so much work involved in adding new dependencies there.